microprofile.ls/com.redhat.microprofile.ls/pom.xml

```diff
@@ -7,7 +7,30 @@
 	<version>0.0.6-SNAPSHOT</version>
 
 	<name>MicroProfile Language Server</name>
+	<description>MicroProfile Language Server</description>
 	<url>http://maven.apache.org</url>
```

Comment:
@fbricon what is this url?

Reply:
yes absolutely

Reply:
well no, https://github.com/redhat-developer/quarkus-ls for now

Reply:
@fbricon thanks for your feedback. @jeffmaury could you change that please in your PR please.